backend/src/main/java/fk/prof/backend/util/ZookeeperUtil.java

```diff
@@ -2,15 +2,27 @@
 
 import io.vertx.core.Future;
 import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.utils.ZKPaths;
+import org.apache.zookeeper.CreateMode;
 import org.apache.zookeeper.KeeperException;
 
+import java.util.List;
+
 public class ZookeeperUtil {
 
+  private static final String DELIMITER = "/";
```

Comment:
maybe make this public and use it in policystore rather than declaring delimiter in that class again